src/wallet/rpcdump.cpp

```diff
@@ -970,13 +984,16 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
             "  [     (array of json objects)\n"
             "    {\n"
             "      \"scriptPubKey\": \"<script>\" | { \"address\":\"<address>\" }, (string / json, required) Type of scriptPubKey (string for script, json for address)\n"
+            "      \"timestamp\": 1454686740 | \"now\"                       , (integer / string) Integer timestamp, or the string \"now\" to substitute the current synced\n"
```

Comment:
Updated in af06509f9c0bb92231ab41fd5fd9e89106b65b0e. Personally I think `"timestamp": 1454686740` is more informative than `"timestamp": <timestamp>`, made the requested change anyway.